app/client/components/ChartView.ts

```diff
@@ -846,7 +846,7 @@ export class ChartConfig extends GrainJSDisposable {
     return cssFieldEntry(
       cssFieldLabel(dom.text(col.label)),
       cssRemoveIcon(
-        'Remove',
+        t('Remove'),
```

Comment:
I admit I have never seen what the l10n side of this would look like. Will translators have enough context to know what is being removed here?